parser.y

```diff
@@ -4884,6 +4885,7 @@ UnReservedKeyword:
 |	"ADVISE"
 |	"ASCII"
 |	"AUTO_INCREMENT"
+|	"AUTO_INCREMENT_CACHE"
```

Comment:
Please rename it to `AUTO_ID_CACHE`, as you mentioned, it doesn't affect AutoIncrement column only.